AI/BattleAI/common.h

```diff
@@ -11,8 +11,7 @@
 
 class CBattleCallback;
 
-template<typename Key, typename Val, typename Val2>
-const Val getValOr(const std::map<Key, Val> &Map, const Key &key, const Val2 defaultValue)
+template<typename Key, typename Val, typename Val2> const Val getValOr(const std::map<Key, Val> & Map, const Key & key, const Val2 defaultValue)
```

Comment:
I would say... always use new line. Will be easier to parse in head then as well.